knowledge-base/how-to-use-kendodatetimepicker-as-an-editor-in-the-html5-viewer-s-parameters-area.md

```diff
@@ -0,0 +1,124 @@
+---
+title: How to use kendoDateTimePicker as an editor in the HTML5 Viewer's Parameters Area
+description: A frequently asked question is how to replace the default kendoDatePicker with kendoDateTimePicker widget.
```

Comment:
Avoid fluff. Change to: "How to replace the default kendoDatePicker with kendoDateTimePicker widget."